Tasks/Gradle/task.loc.json

```diff
@@ -268,6 +319,9 @@
     },
     {
       "name": "sqDbDetailsRequired",
+      "aliases": [
+        "sonarQubeSpecifyDb"
```

Comment:
At least in C#, the rule is that 2-letter acronyms should not be camel-cased, so this would be `sonarQubeSpecifyDB`

Reply:
:+1: